packages/adaptive_scaffold/example/lib/adaptive_layout_demo.dart

```diff
@@ -54,54 +54,200 @@ class MyHomePage extends StatelessWidget {
+            builder: (_) => AdaptiveScaffold.toRailFromDestinations(
+              extended: true,
+              leading: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceAround,
+                children: const [
+                  Text(
+                    "REPLY",
+                    style: TextStyle(color: Color.fromARGB(255, 255, 201, 197)),
+                  ),
+                  Icon(Icons.menu_open)
+                ],
+              ),
+              destinations: destinations,
+              trailing: trailingNavRail,
+            ),
           ),
         },
       ),
-      // Body switches between a ListView and a GridView from small to medium
-      // breakpoints and onwards.
       body: SlotLayout(
-        config: <Breakpoint, SlotLayoutConfig?>{
+        config: <Breakpoint, SlotLayoutConfig>{
           Breakpoints.small: SlotLayout.from(
             key: const Key('body'),
             builder: (_) => ListView.builder(
-                itemCount: 10, itemBuilder: (_, int idx) => children[idx]),
+              itemCount: 10,
+              itemBuilder: (context, index) => Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: Container(
+                  color: const Color.fromARGB(255, 255, 201, 197),
+                  height: 400,
+                ),
+              ),
+            ),
           ),
-          Breakpoints.mediumAndUp: SlotLayout.from(
+          Breakpoints.medium: SlotLayout.from(
+            key: const Key('body1'),
+            builder: (_) =>
+                GridView.count(crossAxisCount: 2, children: <Widget>[
+              for (int i = 0; i < 10; i++)
+                Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Container(
+                    color: const Color.fromARGB(255, 255, 201, 197),
+                    height: 400,
+                  ),
+                )
+            ]),
+          ),
+          Breakpoints.large: SlotLayout.from(
             key: const Key('body1'),
             builder: (_) =>
-                GridView.count(crossAxisCount: 2, children: children),
+                GridView.count(crossAxisCount: 2, children: <Widget>[
+              for (int i = 0; i < 10; i++)
+                Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Container(
+                    color: const Color.fromARGB(255, 255, 201, 197),
+                    height: 400,
+                  ),
+                )
+            ]),
           ),
         },
       ),
-      // BottomNavigation is only active in small views defined as under 600 dp
-      // width.
       bottomNavigation: SlotLayout(
-        config: <Breakpoint, SlotLayoutConfig?>{
+        config: {
           Breakpoints.small: SlotLayout.from(
             key: const Key('bn'),
             inAnimation: AdaptiveScaffold.bottomToTop,
+            outAnimation: AdaptiveScaffold.topToBottom,
```

Comment:
You can delete the outAnimation since the default movement off the screen mimics the animation.